events/static/css/fullcalendar.print.css

```diff
@@ -0,0 +1,208 @@
+/*!
```

Comment:
This library was in cdn. For comfortable development and not be depend on Internet I  decided relocate all static to local.